src/Tests/dotnet-workload-install.Tests/MockWorkloadManifestUpdater.cs

```diff
@@ -2,23 +2,29 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Collections.Generic;
+using System.IO;
 using System.Threading.Tasks;
 using Microsoft.DotNet.Workloads.Workload.Install;
+using Microsoft.Extensions.EnvironmentAbstractions;
 
 namespace Microsoft.DotNet.Cli.Workload.Install.Tests
 {
     internal class MockWorkloadManifestUpdater : IWorkloadManifestUpdater
     {
         public int UpdateAdvertisingManifestsCallCount = 0;
         public int CalculateManifestUpdatesCallCount = 0;
+        public int DownloadManifestPackagesCallCount = 0;
```

Comment:
nit: Good posts on mock and some solution to reduce them http://arlobelshee.com/tag/no-mocks/